.gitignore

```diff
@@ -13,4 +13,4 @@ tmp/
 # Default Waypoint database file
 data.db
 
-.vscode
+# .vscode - commented for customization
```

Comment:
Could we elaborate on this comment? Perhaps “We intentionally commit .vscode in order to share project-specific settings”

Reply:
Yes definitely!